docs/usability/using_extdep.md

```diff
@@ -91,6 +91,7 @@ Ext_Dep Example json file
 - source: see per type
 - version: see per type
 - flags: Optional conditions that can be applied. Can be empty list
+- error_msg: an optional message that will disabled to the user if this ext_dep fails to fetch
```

Comment:
this should be moved down to the optional region.  The field is optional.

Reply:
Thanks